package.json

```diff
@@ -15,7 +15,12 @@
     "htmlparser2": "3.8.3",
     "cssstyle": "0.2.29",
     "webpack": "1.12.2",
-    "webpack-dev-server": "1.12.1"
+    "webpack-dev-server": "1.12.1",
+    "uglify-js": "2.6.1",
+    "bufferutil": "1.2.1",
+    "utf-8-validate": "1.2.1",
+    "tls": "0.0.1",
+    "fs": "0.0.2"
```

Comment:
Hmm, not sure I understand what's going on here.  Several of those, e.g. uglify-js and bufferutil did get installed by 
"npm install" on one of my Linux systems, but tls and fs are not.  Either way these should be recursive dependencies that we don't need to specify explicitly...